backoffice/locale/fr/LC_MESSAGES/django.po

```diff
@@ -18,95 +18,226 @@ msgstr ""
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:70
+msgid "Previous Certificate Tasks"
+msgstr "Historique des générations d'attestations"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:39
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:53
 msgid "Published"
 msgstr "Publié"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:28
-msgid "Running dates"
-msgstr "Dates de déroulement"
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:49
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:88
+msgid "Sample"
+msgstr "Exemple"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/courses.html:41
+msgid "Start"
+msgstr "Début"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:87
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:175
 msgid "Students enrolled"
 msgstr "Utilisateurs inscrits"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:33
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:63
 msgid "Students enrolled in this course: "
 msgstr "Utilisateurs inscrits à ce cours : "
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/base.html:13
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/base.html:44
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:37
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:77
+msgid "Students with certification"
+msgstr "Apprenants ayant obtenu l'attestation"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:38
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:78
+msgid "Students without certification"
+msgstr "Apprenants n'ayant pas obtenu l'attestation"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/base.html:11
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/base.html:32
 msgid "Support team"
 msgstr "Backoffice cellule d'appui"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:30
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:40
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/certificate.html:80
+msgid "Task Start"
+msgstr "Date de lancement"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:48
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:109
 msgid "Teachers"
 msgstr "Professeurs"
 
-#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:57
+#: /edx/app/edxapp/fun-apps/backoffice/views.py:101
+msgid "Teachers have been updated"
+msgstr "La liste de professeurs a été mise à jour"
+
+#: /edx/app/edxapp/fun-apps/backoffice/templates/backoffice/course.html:89
 msgid "This course do not belongs to any university"
```

Comment:
msgid "This course do not belong to any university"

(désolé Julien, je sais que ce n'est pas toi qui a écrit cette trad)